packages/labs/context/src/lib/value-notifier.ts

```diff
@@ -45,26 +55,65 @@ export class ValueNotifier<T> {
   }
 
   updateObservers = (): void => {
-    for (const [callback, disposer] of this.disposers) {
+    for (const [callback, {disposer}] of this.callbacks) {
       callback(this._value, disposer);
     }
   };
 
-  addCallback(callback: ContextCallback<T>, subscribe?: boolean): void {
+  addCallback(
+    callback: ContextCallback<T>,
+    subscribe?: boolean,
+    consumerHost?: Element
```

Comment:
Looking through this change – won't `consumerHost` always be an Element? Can the type signature be updated to reflect that?

Reply:
That would make this a breaking change. Maybe we want that though?

Reply:
pre-graduation is a good time for some breaking changes, although I guess it can be in a separate change (if at all).

Reply:
Good point! Done